openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -46,13 +49,18 @@ def _get_annotation(adata):
 def _atac_genes_score(adata, top_genes=500, threshold=1):
     import pybedtools
 
+    # get annotation for TSS
+    _get_annotation(adata)
     # basic quality control
     sc.pp.filter_cells(adata, min_genes=200)
     sc.pp.filter_genes(adata, min_cells=5)
 
-    adata.var["mt"] = adata.var.gene_short_name.str.startswith(
+    adata.var["mt"] = adata.var.iloc[:, 2].str.startswith(
```

Comment:
This is highly unstable. What if the gene short name is not in the 3rd column?

Reply:
I correct this part with gene_short_name, and add assertion in the checks.py